params/denomination.go

```diff
@@ -21,15 +21,14 @@
 package params
 
 // These are the multipliers for KAIA denominations.
-// Example: To get the peb value of an amount in 'ston', use
+// Example: To get the wei value of an amount in 'gwei', use
 //
-//	new(big.Int).Mul(value, big.NewInt(params.Ston))
+//	new(big.Int).Mul(value, big.NewInt(params.Gwei))
 const (
-	Peb      = 1    // official notation 'peb'
-	Kpeb     = 1e3  // official notation 'kpeb'
-	Mpeb     = 1e6  // same
-	Gpeb     = 1e9  // same
-	Ston     = 1e9  // official notation 'ston'
+	Wei      = 1    // official notation 'wei'
```

Comment:
You can delete the unused variables such as UKAIA. c,f. https://github.com/ethereum/go-ethereum/blob/v1.14.3/params/denomination.go

Reply:
fixed eae0f459